src/js/helpers.js

```diff
@@ -92,52 +92,52 @@ export function createPageList(formConfig, formPages) {
     });
 }
 
-/*
- * Create the routes based on a form config. This goes through each chapter in a form
- * config, pulls out the config for each page, then generates a list of Route components with the
- * config as props
- */
-export function createRoutes(formConfig) {
-  const formPages = createFormPageList(formConfig);
-  const pageList = createPageList(formConfig, formPages);
-  let routes = formPages
-    .map(page => {
-      return {
-        path: page.path,
-        component: page.component || FormPage,
-        pageConfig: page,
-        pageList,
-        urlPrefix: formConfig.urlPrefix
-      };
-    });
-  if (formConfig.introduction) {
-    routes = [
-      {
-        path: 'introduction',
-        component: formConfig.introduction,
-        formConfig,
-        pageList
-      }
-    ].concat(routes);
-  }
-
-  return routes.concat([
-    {
-      path: 'review-and-submit',
-      formConfig,
-      component: ReviewPage,
-      pageList
-    },
-    {
-      path: 'confirmation',
-      component: formConfig.confirmation
-    },
-    {
-      path: '*',
-      onEnter: (nextState, replace) => replace(formConfig.urlPrefix || '/')
-    }
-  ]);
-}
+// /*
+//  * Create the routes based on a form config. This goes through each chapter in a form
+//  * config, pulls out the config for each page, then generates a list of Route components with the
+//  * config as props
+//  */
+// export function createRoutes(formConfig) {
```

Comment:
Copied this function into the starter app and tested using it there to create the routes to make sure everything still works with this pulled out.